SpringProject/src/main/java/com/gradproject2019/users/service/UserServiceImpl.java

```diff
@@ -64,6 +64,14 @@ public UserResponseDto editUser(UUID token, Long userId, UserPatchRequestDto use
         return new UserResponseDto().from(getUserById(userId));
     }
 
+    @Override
+    public List<UserResponseDto> findUserByManagerId(UUID token, Long managerId) {
```

Comment:
Minor point but as this is returning a list of users, probably `findUsers` would be a better prefix

Reply:
changed :)
i like the minor points, they are easier to rectify haha